lib/codegen/meta-python/base/instructions.py

```diff
@@ -51,6 +51,7 @@
 EBB = Operand('EBB', entities.ebb, doc='Destination extended basic block')
 args = Operand('args', VARIABLE_ARGS, doc='EBB arguments')
 
+
```

Comment:
Why this extra newline?

Reply:
This is a WIP. I’ll clean up all those things toward the end.